src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DictionaryGlobals.cs

```diff
@@ -76,6 +77,8 @@ static DictionaryGlobals()
                 RefLocalName = dictionary.Add(Globals.RefLocalName);
                 ArraySizeLocalName = dictionary.Add(Globals.ArraySizeLocalName);
                 EmptyString = dictionary.Add(String.Empty);
+                ISerializableFactoryTypeLocalName = dictionary.Add(Globals.ISerializableFactoryTypeLocalName);
```

Comment:
Commenting here as I can't comment on the dictionary constructor line.  
The dictionary size doesn't match the number of elements below. We should either add the missing dictionary items back or reduce this number.